examples/formbot/data/stories.md

```diff
@@ -1,4 +1,6 @@
 ## happy path
+* greet
+	- utter_greet
```

Comment:
can we make sure the indents for the `utter_greet` align with the rest of the actions?

Reply:
Oh, good catch. Fixed now.